apps/sage_intacct/connector.py

```
@@ -1361,6 +1361,58 @@ def get_expense_reports(self, expense_report_ids: list[str], fields: list = None
+    def get_accounting_fields(self, resource_type: str) -> dict:
+        """
+        Retrieve accounting fields for a specific resource type using REST API.
+        This is for internal API use only.
+        :param resource_type: The type of resource to fetch accounting fields for.
+        :return: Parsed JSON representation of the accounting fields.
+        """
+        try:
+            fields = self.get_field_list(resource_type)
+            module = getattr(self.connection, resource_type)
+        except AttributeError:
+            raise ValueError(f"Resource type '{resource_type}' is not supported")
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)